tests/core/manifest/TemplateLoaderTest.php

```diff
@@ -26,6 +26,8 @@ public function setUp() {
 	 * Test that 'main' and 'Layout' templates are loaded from module
 	 */
 	public function testFindTemplatesInModule() {
```

Comment:
Yeah, the whole file needs replacing with a different one that tests the new behaviour. Change to this PR with better docs & tests should be up today.